crosswalks/Citation File Format 1.2.0.csv

```diff
@@ -1,76 +1,68 @@
-Property,Citation File Format Core (CFF-Core) 1.0.2
-codeRepository,repository-code
-programmingLanguage,
-runtimePlatform,
-targetProduct,
-applicationCategory,
-applicationSubCategory,
-downloadUrl,repository-artifact
-fileSize,
-installUrl,
-memoryRequirements,
-operatingSystem,
-permissions,
-processorRequirements,
-releaseNotes,
-softwareHelp,
-softwareRequirements,
-softwareVersion,
-storageRequirements,
-supportingData,
-author,authors
-citation,
-contributor,
-copyrightHolder,
-copyrightYear,
-dateCreated,
-dateModified,
-datePublished,date-released
-editor,
-encoding,
-fileFormat,
-funder,
-keywords,keywords
-license,license/license-url
-producer,
-provider,
-publisher,
-sponsor,
-version,version
-isAccessibleForFree,
-isPartOf,references
-hasPart,
-position,
-description,abstract
-identifier,doi
-name,title
-sameAs,
-url,url
-relatedLink,
-givenName,person.given-names
-familyName,person.name-particle + person.family-names + person.name-suffix
-email,person.email/entity.email
-affiliation,person.affiliation
-identifier,person.orcid / entity.orcid
-name,entity.name
-address,person.address + person.city + person.region + person.post-code + person.country / entity.address + entity.city + entity.region + entity.post-code + entity.country
-type,reference.type
-id,
-softwareSuggestions,
-maintainer,
-contIntegration,
-buildInstructions,
-developmentStatus,
-embargoDate,
-funding,
-issueTracker,repository
-referencePublication,references
-readme,
-,
-,
-,
-,
-,
-,
-,
-,
+Property,Citation File Format (1.2.0)
+codeRepository,repository-code
+programmingLanguage,
+runtimePlatform,
+targetProduct,
+applicationCategory,
+applicationSubCategory,
+downloadUrl,repository-artifact
+fileSize,
+installUrl,
+memoryRequirements,
+operatingSystem,
+permissions,
+processorRequirements,
+releaseNotes,
+softwareHelp,
+softwareRequirements,references
+softwareVersion,version
+storageRequirements,
+supportingData,
+author,authors
+citation,Metadata in the CFF root document / preferred-citation / references
+contributor,
```

Comment:
Consider putting `authors` here since in CFF 1.2.0 `authors` contains contributors.

Reply:
I'll put it there, but leave it in brackets, as there is a qualitative difference between authors and contributors, as not all contributors may qualify as authors; and depending on how you interpret "contributors" (i.e., as people that push commits vs. people that contribute in other, perhaps non-tangible ways), not all authors are contributors.

I'll also add a respective section in the README.

Reply:
Done in ba11bfaade892b738bf86f9546660f7adae7946b.